packages/react-notion-x/src/block.tsx

```diff
@@ -794,7 +794,8 @@ export const Block: React.FC<BlockProps> = (props) => {
                 key={column}
                 className={color ? `notion-${color}` : ''}
                 style={{
-                  width: formatMap?.[column]?.width || 120
```

Comment:
Can `formatMap[column].width` exist?

Reply:
I'm not exactly sure. I know that columns cannot be resized manually. 

Will do one thing. Will keep the width property too, but instead of having the fallback width as `120`, I will keep it as `auto`.